packages/dso-toolkit/src/components/alert/alert.mixins.scss

```diff
@@ -78,14 +78,41 @@
   @media screen and (min-width: #{media-query-breakpoints.$screen-xs-min + 1}) {
     padding-block: alert-variables.$block-padding;
     padding-inline: alert-variables.$inline-start-padding alert-variables.$inline-padding;
+
+    &.dso-small {
+      padding-block: alert-variables.$block-padding * 0.5;
+      padding-inline: alert-variables.$inline-padding;
+    }
   }
 
   @media screen and (max-width: media-query-breakpoints.$screen-xs-min) {
     padding-block: alert-variables.$small-padding-block-start alert-variables.$small-padding;
     padding-inline: alert-variables.$small-padding;
+
+    &.dso-small {
+      padding-block: alert-variables.$block-padding * 0.5;
+      padding-inline: alert-variables.$inline-padding;
+    }
   }
 
   &:not(:first-child) {
     margin-block-start: units.$u3;
   }
 }
+
+@mixin small($bottom: 0, $top: 0) {
```

Comment:
Waarom moet deze mixin parameters accepteren? Ik begrijp dat het met Web Component vs. HTML/CSS te maken heeft, weten we waar dat verschil zit?

Ik heb liever dat dit verschil wordt gladgestreken ipv. dat we eromheen werken.

Reply:
Als gevolg van het verdwijnen van de border-top en -bottom zijn er nu screenshot diffs ontstaan. Ik heb nl. geen 2px extra padding toegevoegd...

Reply:
Mooi werk!